README.md

```diff
@@ -1,70 +1,47 @@
-# Getting Started with Create React App
+# Space Travelers
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+The web app retrieves real data from SpaceX API, and provides commercial and scientific space travel services by allowing users to book rockets and join selected space missions.
 
-## Available Scripts
+## Live Demo
 
-In the project directory, you can run:
+[space-travelers-fbclh.vercel.app](https://space-travelers-fbclh.vercel.app/)
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- React | Redux | React Router | Jest | Testing Library | API
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Getting Started
 
-### `npm test`
+Clone
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+```sh
+   git clone git@github.com:fbclh/space-travelers.git
+```
 
-### `npm run build`
+Run
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+```sh
+   cd space-travelers
+   npm install
+   npm start
+```
 
```

Comment:
- [ ] **Required:** Kindly add the `Testing` section to your readme to specify the procedure for running the tests. This makes your documentation more professional.